nessie/sql_templates/update_rds_indexes_sisedo_full.template.sql

```diff
@@ -185,11 +187,12 @@ AND edo.cs_course_id = 'cms-version-independent-id';
 -- Copy over past-term sections from EDL-sourced sis_sections table.
 
 INSERT INTO {rds_schema_sis_internal}.edo_sections
-(sis_term_id, sis_section_id, is_primary, sis_course_name, sis_course_title, sis_instruction_format,
+(sis_term_id, sis_section_id, is_primary, dept_name, sis_course_name, sis_course_title, sis_instruction_format,
       sis_section_num, cs_course_id, session_code, instruction_mode, primary_associated_section_id,
       instructor_uid, instructor_name, instructor_role_code, meeting_location,
       meeting_days, meeting_start_time, meeting_end_time, meeting_start_date, meeting_end_date)
-SELECT sis_term_id, sis_section_id, is_primary, sis_course_name, sis_course_title, sis_instruction_format,
+SELECT sis_term_id, sis_section_id, is_primary, split_part(sis_course_name, ' ', 1)  as dept_name,
```

Comment:
@pauline2k - would the `split_part` logic work on, for example, sis_course_name = "NUC ENG 124"?

Reply:
@johncrossman sis_course_name in this source table is represented as 'NUCENG', so the split will work. Over in edo_sections (which we're using for current sections) the department is 'NUC ENG', matching bCourses subaccounts. So there will be an inconsistency between present-term and past-term data, but since we never provision course sites out of past-term data it shouldn't affect workflow.